projects/ng-rule-editor/src/lib/rule-editor.service.spec.ts

```diff
@@ -17,46 +21,46 @@ describe('RuleEditorService', () => {
   });
 
   it("should return scored questionnaire with var1", () => {
-    expect(service.addTotalScoreRule(mockdata.fhir, mockdata.linkId)).to.include(mockdata.var1);
+    expect(service.addTotalScoreRule(fhir.phq9, phq9ScoreLinkId)).to.include(insert(phq9Vars[0]));
```

Comment:
The "it"s for 1-8 could be declared in a loop.

Reply:
Fixed